src/components/organisms/TimingSpace/TimingSpace.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useCallback, useMemo } from "react";
+
+import { VenueEvent } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import { useShowHide } from "hooks/useShowHide";
+
+import { TimingEvent } from "components/organisms/TimingEvent";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import "./TimingSpace.scss";
+
+export type TimingSpaceProps = {
+  spaceName: string;
+  spaceEvents: WithId<VenueEvent>[];
+  setEditedEvent: React.Dispatch<
+    React.SetStateAction<WithId<VenueEvent> | undefined>
+  >;
+  setShowCreateEventModal: () => void;
+};
+
+export const TimingSpace: React.FC<TimingSpaceProps> = ({
+  spaceName,
+  spaceEvents,
+  setShowCreateEventModal,
+  setEditedEvent,
+}) => {
+  const { isShown: selectedSpace, toggle: toggleSelectedSpace } = useShowHide();
+
+  const handleCreateButtonClick = useCallback(
+    (e) => {
+      e.stopPropagation();
+
+      setShowCreateEventModal();
```

Comment:
onClickCreateButton()

Reply:
fixed in 694c85c4c23d817716346efdbc1df16c7b903d58